packages/core/src/RenderPipeline/DepthOnlyPass.ts

```diff
@@ -56,8 +57,8 @@ export class DepthOnlyPass extends PipelinePass {
     rhi.viewport(0, 0, renderTarget.width, renderTarget.height);
     rhi.scissor(0, 0, renderTarget.width, renderTarget.height);
 
-    cullingResults.opaqueQueue.render(camera, Layer.Everything, PipelineStage.DepthOnly);
-    cullingResults.alphaTestQueue.render(camera, Layer.Everything, PipelineStage.DepthOnly);
+    cullingResults.opaqueQueue.render(camera, PipelineStage.DepthOnly);
```

Comment:
`clearRenderTarget` here after viewport and scissor

Reply:
You mean clearRenderTarget should after viewport and scissor?